docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,42 @@
+# Prerequisites
+1. Access to Heroku `discord-soc-bot` app
+2. Heroku git remote setup (see [docs](https://devcenter.heroku.com/articles/git))
+3. GitHub repository write access
+
+# Steps
+1. Merge all PRs you wish to deploy into `develop` branch
+2. Fetch and checkout `develop` branch
+3. Find production `DATABASE_URL` in Heroku
+4. Make a backup of the DB just in case anything goes wrong
+```
+pg_dump [DATABASE_URL] > dbBackup[DATE].sql
+```
+e.g.
+```
+pg_dump postgresql://discordbot:somePassword@localhost/discord_bot > dbBackup20211109.sql
+```
+5. Change `yoyo-local.ini` `database` to the `DATABASE_URL` from Heroku
+6. Run yoyo migrations
```

Comment:
I would add - "After this command carefully check logs if everything went ok. Also connect to DB if you can and check if DB tables were updated as you wanted them"